test/grouping.jl

```diff
@@ -2,14 +2,97 @@ module TestGrouping
     using Base.Test
     using DataTables
 
+    srand(1)
     dt = DataTable(a = repeat([1, 2, 3, 4], outer=[2]),
                    b = repeat([2, 1], outer=[4]),
                    c = randn(8))
     #dt[6, :a] = Nullable()
     #dt[7, :b] = Nullable()
 
-    cols = [:a, :b]
+    nullfree = DataTable(Any[collect(1:10)], [:x1])
+    # colwise(::Vector{<:Function}, ::AbstractDataTable)
+    fxn = [sum]
```

Comment:
Yeah, that was just a minor stylistic thing. But indeed using testsets is the best solution.